### PR TITLE
Fixed bash formatting issue

### PR DESCRIPTION
**Pull request by anantasty**

### CODE REVIEW

File: scripts/create_additive_authoritative_structures.sh [hunk 1]
```diff
@@ -150,7 +150,7 @@ function main {
   then
     build_authoritative
   fi
-  echo "$OUTPUT"| sed -e 's/.*/{"data": "&/; s/$/"}/'
+  jq -n --arg output "$OUTPUT" '{data:$output}'
```
Comment (morgante) [inline scripts/create_additive_authoritative_structures.sh]: Is there a way to do this without jq?

File: test/fixtures/full/iam.tf [hunk 2]
```diff
@@ -1,120 +0,0 @@
-/**
```
Comment (morgante) [inline test/fixtures/full/iam.tf]: Why are we deleting this file? It seems unrelated.

File: scripts/create_additive_authoritative_structures.sh (hunk 1 above)
Comment (anantasty) [reply scripts/create_additive_authoritative_structures.sh]: this could most likely be done with sed.

**Comment by paulpalamarchuk** [comment]
@morgante FYI, in [PR-43](https://github.com/terraform-google-modules/terraform-google-iam/pull/43) bash-script files removed, as structure formatting made in TF-submodules
